C_sharp_exercises_2/C_sharp_exercises_2/Exercises4.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+//Exercise 4:
+//Create a class Book which should have author, title and createdDate properties.
+//Create 10 instances of Book class.
+//1.Filter latest 5 created Books.
+//2. Group the books which authors are same person
+//3. Find Books which title starts with A letter
+namespace C_sharp_exercises_2
+{
+    internal class Exercises4
+    {
+        public string RetD()
```

Comment:
RetD is not a proper name, instead of creating date time here do it in constructor